docs/api/ref/GetAgent.md

```diff
@@ -0,0 +1,224 @@
+
+If no Desktop Agent is found, a failover function may be supplied by app allowing it to start or otherwise connect to a Desktop Agent (e.g. by loading a proprietary adaptor that returns a `DesktopAgent` implementation or by creating a window or iframe of its own that will provide a Desktop Agent Proxy).
+
+The definition of the `getAgent()` function is as follows:
+
+```ts
+type GetAgentType = ( 
+  params?: GetAgentParams,  
+) => Promise<DesktopAgent>; 
+```
+
+A small number of arguments are accepted that can affect the behavior of `getAgent` and to provide a fallback in case a Desktop Agent is not found, allowing the application to start its own agent or use another mechanism (such a proprietary adaptor) to connect to one.
+
+```ts
+/** 
+ * @typedef {Object} GetAgentParams Type representing parameters passed to the 
+ * getAgent function. 
+ * 
+ * @property {string} identityUrl The app's current URL is normally sent to 
+ * a web-based desktop agent to help establish its identity. This property 
+ * may be used to override the URL sent (to handle situations where an app's 
+ * URL is not sufficiently stable to use for identity purposes). The URL set 
+ * MUST match the origin of the application (scheme, hostname, and port) or  
+ * it will be ignored. If not specified, the app's current URL will be used.  
+ * 
+ * @property {number} timeout Number of milliseconds to allow for an fdc3  
+ * implementation to be found before calling the failover function or 
+ * rejecting (default 750). Note that the timeout is cancelled as soon as a 
+ * Desktop Agent is detected. There may be additional set-up steps to perform 
+ * which will happen outside the timeout. 
+ * 
+ * @property {boolean} channelSelector Flag indicating that the application  
+ * needs access to a channel selector UI (i.e. because it supports User Channels 
+ * and does not implement its own UI for selecting channels). Defaults to true. 
+ * MAY be ignored by Desktop Agent Preload (container) implementations. 
+ * 
+ * @property {boolean} intentResolver Flag indicating that the application  
+ * needs access to an intent resolver UI (i.e. because it supports raising one
+ * or more intents and and does not implement its own UI for selecting target 
+ * apps. Default to `true`. MAY be ignored by Desktop Agent Preload (container) 
+ * implementations. 
+ * 
+ * @property {boolean} dontSetWindowFdc3 For backwards compatibility, `getAgent` 
+ * will set a reference to the Desktop Agent implementation at `window.fdc3` 
+ * if one does not already exist, and will fire the `fdc3Ready` event. Setting  
```

Comment:
For clarity and for consistency with the descriptions of the other properties above, it would be good to add:

"Defaults to `false`."

immediately prior to the final sentence of this description.

Reply:
Fixed in be131b655bbc4812c9cfec080fb8ee86ac81cb85